cmd/node/main.go

```diff
@@ -353,22 +365,32 @@ func startNode(ctx *cli.Context, log *logger.Logger) error {
 		return err
 	}
 
-	publicKey, err := pubKey.ToByteArray()
-	if err != nil {
-		return err
-	}
+	workingDir := ctx.GlobalString(workingDirectory.Name)
 
-	uniqueID = core.GetTrimmedPk(hex.EncodeToString(publicKey))
+	uniqueDBFolder := filepath.Join(workingDir, defaultDBPath, defaultEpochString+fmt.Sprint(0),
```

Comment:
this can be changed to something like:
```GO
fmt.Sprintf("%s%d", defaultEpochString,0)
```

also the next line

Reply:
fixed